### ISSUE TITLE
Misspellings placed after hyperlinks are not indicated

### ISSUE THREAD

**Issue by cstrahan** [inkdropapp/inkdrop-spell-checker#7]
Take the following markup for example:

```
[example](http://example.com) foobar
```

"foobar" has no visual indication as a spelling error. This effect seems to carry on for the remainder of the surrounding paragraph / bullet point.

For context, I have no other plugins installed, and I just now (about 5 minutes ago) installed this plugin.

**Comment by cstrahan** [comment]
This is the first time I've reached out, and it feels a little wrong to just dryly point out an issue, so I'd like to take the opportunity to say:

Thanks for making Inkdrop, @craftzdog! It has been an incredible tool in getting my notes written down in a cross platform way, without the clunky overhead of WYSIWYG text editing. For a while now, I have kept a loose pile of markdown notes that I'd manually commit to a git repo, and I kept telling myself I'd put together some sort of read-only wiki style frontend in front of it... but I'd never get around to it. Inkdrop is pretty much exactly what I've wanted all along, so I now use it for practically everything I wish to take notes on. And I do a lot more note taking now that the experience is so great, so now many aspects of my life (both technical and non-technical) are a lot more organized as a result. I'm also really looking forward to using the support for LaTeX and diagrams (swim-lanes and such) in the future.
